src/librustc/infer/error_reporting/nice_region_error/static_impl_trait.rs

```diff
@@ -54,7 +54,7 @@ impl<'a, 'tcx> NiceRegionError<'a, 'tcx> {
                             err.span_suggestion(
                                 fn_return_span,
                                 &format!(
-                                    "you can add a constraint to the return type to make it last \
+                                    "you can add a bound to the return type to make it last \
                                  less than `'static` and match {}",
```

Comment:
```
match {}
```
can be replaced with
```
match `{}`
```
to match already applied suggestion to `add_lifetime_bound_suggestion_to_diagnostic` method